src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/FalconElbow/FalconElbow.java

```diff
@@ -46,17 +46,22 @@ public void resetAngle(Rotation2d position) {
     @Override
     public void moveToAngle(Rotation2d targetAngle) {
         motor.setControl(
-                motionMagicDutyCycle.withPosition(targetAngle.getRadians() / ElbowConstants.RELATIVE_POSITION_CONVERSION_FACTOR)
-                .withFeedForward(FalconElbowConstants.SIMPLE_MOTOR_FEED_FORWARD.calculate(motor.getVelocity().getValue() * ElbowConstants.RELATIVE_VELOCITY_CONVERSION_FACTOR )));
+                motionMagicDutyCycle.withPosition(targetAngle.getRadians() / ElbowConstants.CONVERSION_FACTOR)
+                .withFeedForward(FalconElbowConstants.SIMPLE_MOTOR_FEED_FORWARD.calculate(motor.getVelocity().getValue() * ElbowConstants.CONVERSION_FACTOR )));
+    }
+
+    @Override
+    public void standInPlace() {
```

Comment:
we should add a pid controller to stay in the same location

Reply:
same